frontend/src/pages/GraphicalSelector/GraphicalSelector.tsx

```diff
@@ -200,6 +207,25 @@ const GraphicalSelector = () => {
     }
   };
 
+  function getZoomRatio() {
+    if (!graph) return 0;
```

Comment:
`return;` over `return 0;`

Reply:
there are two problesm with that:
1. the return type must be an integer as the function returns must be a consistent type
2. Since we add it to the funcs zoomIn and zoomOut it would return an undefined value and we cannot not add that to zoom `graph.zoomTo(zoom + getZoomRatio());`

Reply:
Oh right me dum dum xd. I legit didn't see it was returning a number before whoops

Reply:
Thanks for the clarifications :D

Reply:
haha all g thx to for the merge :D